src/os/feature/feature.js

```diff
@@ -1380,6 +1381,43 @@ const validateGeometries = function(feature, opt_quiet) {
   return count;
 };
 
+/**
+ * Validation helper for polygon types.
+ *
+ * MultiPolygon geometries have their internal polygons individually validated, and removed if they fail the validation.
+ * Otherwise the structure of the MultiPolygon is not affected.
+ *
+ * @param {Geometry} geometry The polygon-type geometry to validate
+ * @param {boolean=} opt_quiet If alerts should be suppressed
+ * @return {Geometry|undefined} a valid polygon, or undefined if invalid
+ * @private
+ */
+const validatePolygonType_ = function(geometry, opt_quiet) {
```

Comment:
Module-local symbols should not use `@private` (this is assumed unless the symbol is exported), or a trailing underscore.

https://google.github.io/styleguide/jsguide.html#file-goog-module-exports